cheatsheets/TLS_Cipher_String_Cheat_Sheet.md

```diff
@@ -18,7 +18,7 @@ The recommended cipher strings are based on different scenarios:
 
 **OWASP Cipher String 'A'** (Advanced, wide browser compatibility, e.g. to most newer browser versions):
 
-* Recommended if you control the server and the clients (e.g. by approvement), make sure to check the compatibility before using it.
+* Recommended if you control the server and the clients (e.g. by approval), make sure to check the compatibility before using it.
```

Comment:
I'm not sure about this correction. Again, I suspect this was composed in French.

Reply:
This change reads better, but either way I'm not really sure the brackets add anything. Maybe just remove them, and replace the comma after them with a full stop?